core/src/main/java/org/apache/iceberg/CatalogProperties.java

```diff
@@ -53,6 +56,57 @@ private CatalogProperties() {
   public static final long CACHE_EXPIRATION_INTERVAL_MS_DEFAULT = TimeUnit.SECONDS.toMillis(30);
   public static final long CACHE_EXPIRATION_INTERVAL_MS_OFF = -1;
 
+  /**
+   * When using {@link CachingFileIO} as io-impl, define the internal FileIO that is
+   * wrapped by CachingFileIO.
+   * <p>
+   * Default to {@link HadoopFileIO} if not defined.
+   */
+  public static final String CONTENT_CACHE_WRAPPED_FILE_IO_IMPL = "cache.content.wrapped-io-impl";
+  public static final String CONTENT_CACHE_WRAPPED_FILE_IO_IMPL_DEFAULT = HadoopFileIO.class.getName();
+
+  /**
+   * Controls the duration for which entries in the {@link CachingFileIO} are cached.
+   * <p>
+   * Behavior of specific values of cache.expiration-interval-ms:
+   * <ul>
+   *   <li> Negative Values - Caching disabled</li>
+   *   <li> Zero - Cache entries expires only if it gets evicted due to memory pressure from
+   *        {@link #CONTENT_CACHE_MAX_TOTAL_BYTES} setting.
+   *   </li>
+   *   <li> Positive Values - Cache entries expire if not accessed via the cache after this many milliseconds</li>
+   * </ul>
+   */
+  public static final String CONTENT_CACHE_EXPIRATION_INTERVAL_MS = "cache.content.expiration-interval-ms";
```

Comment:
`content` seems unnecessary. What else would be cached?

Reply:
I see catalog properties already have key `CACHE_ENABLED` and `CACHE_EXPIRATION_INTERVAL_MS` tied to `CachingCatalog` (which has `Cache<TableIdentifier, Table>` inside). Thus, I prefix this with `CONTENT_` to signify that this is different kind of caching. It is caching the file content (bytes) and not file handle or other java objects.

Reply:
I renamed these with `IO_CACHE_` and `io.cache.` prefix to match with `io.cache-enabled`.